CONTRIBUTING.md

```diff
@@ -179,14 +179,20 @@ Cluster API maintains the most recent release/releases for all supported API and
 
 We are going to remove the apiVersions in upcoming releases:
 * v1.5:
-  * Kubernetes API server will stop serving the v1alpha3 apiVersion
+  * Kubernetes API server has stopped serving the v1alpha3 apiVersion
```

Comment:
just a nit, wondering if we can phrase this and the next few in a way that it's neutral to if we already did it or not

Feel free to just resolve

Reply:
Good idea - I've updated the wording now.